app/_hub/kong-inc/mtls-auth/_index.md

```diff
@@ -214,17 +213,19 @@ curl -sX POST https://kong:8001/ca_certificates -F cert=@cert.pem
 
 {% navtab Konnect Cloud %}
 
-Go through the Konnect Manager:
-1. From the Dashboard, select **Shared Config** in the left navigation
-2. Select **Certificates**
-3. Click **New Certificate**
-4. Copy and paste your certificate information and select **Create**
+Go through the Runtime Manager:
+1. In {{site.konnect_short_name}}, click {% konnect_icon runtimes %} **Runtime Manager**. 
```

Comment:
@lena-larionova @Guaris For some reason, this icon wasn't displaying correctly on my localhost, so I just wanted to check that I did this correctly.

Reply:
The icons are in the helium PR, the helium PR was merged into main after this PR was created. You can see how it would look locally by doing a `git rebase main` which should look like this:
![image](https://user-images.githubusercontent.com/23319190/178533117-06f58c8f-9a4d-4421-9099-fa6fae3c425f.png)